nms/app/fbcnms-projects/magmalte/app/theme/default.js

```diff
@@ -214,12 +214,22 @@ export default createMuiTheme({
   typography: {
     ...typography,
   },
+  props: {
+    MuiDialogTitle: {
+      disableTypography: true, // No more wrapping children in typography component for dialog title
+    },
+  },
   overrides: {
     MuiAppBar: {
       colorPrimary: {
         color: colors.primary.white,
       },
     },
+    MuiBackdrop: {
```

Comment:
@ekfuhrmann Does this affect only the dialog ?

Reply:
Yeah the Backdrop is the scrim (dark shading in the back) that occurs when a dialog is up.